src/nunit.analyzers/Extensions/AttributeArgumentSyntaxExtensions.cs

```diff
@@ -22,15 +22,34 @@ internal static class AttributeArgumentSyntaxExtensions
             typeof(DateTime),
         };
 
-        // Intrinsic type converters for types that are not SpecialTypes (and supported in netstandard1.6)
+        // Intrinsic type converters for types
         // https://github.com/dotnet/runtime/blob/master/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs
-        private static readonly List<(Type type, Lazy<TypeConverter> typeConverter)> IntrinsicTypeConverters =
-            new List<(Type type, Lazy<TypeConverter> typeConverter)>
+        // For converters that exists in .NET Standard 1.6 we use the converter.
+        // Otherwise, we assume that that the converter exists and it can convert the value.
```

Comment:
```suggestion
        // For converters that exist in .NET Standard 1.6 we use the converter.
        // Otherwise, we assume that the converter exists and it can convert the value.
```